index.html

```diff
@@ -45,12 +48,13 @@ <h2 class="heading">Activity Feed</h2>
       </div>
 
       <div id="get-started">
-        <img src="img/getStarted.png" id="get-started-img"/>
+        <img src="img/getStarted.png" id="get-started-img" />
         <div id="get-started-description" align="center">
-          <a href="https://app.contrastsecurity.com/Contrast">Visit your organization on Teamserver</a> to onboard some spplications.
+          <a id="visitOrg">Visit your organization on Teamserver</a> to onboard some spplications.
```

Comment:
html ids, at least for the moment, are dashed not camelcased

Reply:
ID changed